src/CircleGraph/CircleGraphCtrl.ts

```diff
@@ -145,27 +145,31 @@ export class CircleGraphCtrl {
   private registerEventHandlers() {
     // Handle messages from the webview
     this._webview.onDidReceiveMessage(message => {
+      switch (message.command) {
+        case MessageDefs.alert:
+          Balloon.error(message.text);
+          return;
+        case MessageDefs.request:
+          this.handleRequest(message.url, message.encoding);
+          return;
+        case MessageDefs.loadmodel:
+          this.handleLoadModel(parseInt(message.offset));  // to number
+          return;
+        case MessageDefs.finishload:
+          this.handleFinishLoad();
+          return;
+        default:
+          break;
```

Comment:
although this change is relocation of existing changes, `default` is added to express default fallthrought of not handled message.